File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Member/DeclarationBillingIntegration.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.ClientCare.Contracts.Entities.Member
{
    public class DeclarationBillingIntegration
    {
        public int DeclarationBillingIntegrationId { get; set; }
        public int DeclarationId { get; set; }
        public decimal Amount { get; set; }
        public DeclarationBillingIntegrationStatusEnum DeclarationBillingIntegrationStatus { get; set; }
        public DeclarationBillingIntegrationTypeEnum DeclarationBillingIntegrationType { get; set; }
        public System.DateTime? BillingProcessedDate { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/declaration-billing-integration.ts
import { DeclarationBillingIntegrationStatusEnum } from '../enums/declaration-billing integration-status.enum';
import { DeclarationBillingIntegrationTypeEnum } from '../enums/declaration-billing integration-type.enum';

export class DeclarationBillingIntegration {
  declarationBillingIntegrationId: number;
  declarationId: number;
  amount: number;
  declarationBillingIntegrationStatus: DeclarationBillingIntegrationStatusEnum;
  declarationBillingIntegrationType: DeclarationBillingIntegrationTypeEnum;
  billingProcessedDate: Date;
}
